lib/WP_Auth0_DBManager.php

```diff
@@ -113,13 +113,7 @@ public function install_db( $version_to_install = null, $app_token = '' ) {
 			$options->set('cache_expiration', 1440);
 
 			// Update Client
-			if (!empty($client_id) && !empty($domain)) {
-				$payload = array(
```

Comment:
This was overwriting existing Application settings, which could have unintended effects.